File: C Programming/Assignments/Assignment 7/Q7.c
//7. Write a program to calculate total salary based on basic salary (DA, TA, HRA conditions).
#include<stdio.h>
void sal(float*);
void sal2(float*);

void main()
{
    float a;
    printf("Enter a Salary :");
    scanf("%f",&a);
    if(a<=5000)
    sal(&a);
    else if(a>5000)
    sal2(&a);
}

void sal(float *salary)
{
    float da,ta,hra;
    da=*salary * 0.10;
    ta=*salary * 0.20;
    hra=*salary * 0.25;
    printf("The total Salary is = %f",da+ta+hra+ *salary);
}

void sal2(float *salary)
{
    float da,ta,hra;
    da=*salary * 0.15;
    ta=*salary * 0.25;
    hra=*salary * 0.30;
    printf("The total Salary is = %f",da+ta+hra+ *salary);
}

File: Java/C TO JAVA/Assignment 1/Q7.java
class Q7
{

public static void main(String args[]) {
	int a=32000;
	double da,ta,hra,tot;
	if(a<=5000) {
		da=a*0.10;
		ta=a*0.20;
		hra=a*0.25;
		tot=da+ta+hra+a;
		System.out.printf("%f,%f,%f",da,ta,hra);
		System.out.printf("\nif basic salary is <=5000 then total Salary=%f",tot);
	} else {
		da=a*0.15;
		ta=a*0.25;
		hra=a*0.30;
		tot=da+ta+hra+a;
		System.out.printf("\n%f,%f,%f",da,ta,hra);
		System.out.printf("if basic salary is %d <=5000 then total Salary=%f",a,tot);
	}
}
}

	